src/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1632,6 +1632,16 @@ elementFormDefault="qualified">
         <xs:documentation><!-- _locID_text="PackageTargetFallback" _locComment="" -->Allows packages using alternative monikers to be referenced in this project, which include older (e.g. dnxcore50, dotnet5.x) and Portable Class Library names.</xs:documentation>
       </xs:annotation>
     </xs:element>
+    <xs:element name="AssetTargetFallback" substitutionGroup="msb:Property">
+      <xs:annotation>
+        <xs:documentation><!-- _locID_text="AssetTargetFallback" _locComment="" -->Can be set to one or more targetframeworks. When determining package compatibility, if the package is not determined to have compatible assets for the project based on the projects targetframework, compatibility will be rechecked using each targetframework from the AssetTargetFramework project of the hosting project.</xs:documentation>
```

Comment:
I know you copy-pasted this from NuGet/docs.microsoft.com-nuget#1264 but I'm going to copyedit it anyway.

```suggestion
        <xs:documentation><!-- _locID_text="AssetTargetFallback" _locComment="" -->Can be set to one or more target framework monikers. When determining package compatibility, if the package does not have compatible assets for the project's real target framework, compatibility will be rechecked using each target framework from the AssetTargetFramework project of the referencing project.</xs:documentation>
```